### DISCUSSION TITLE
Mention item list shows up as html on UI

### DISCUSSION THREAD

**Discussion by mek24** [afconsult/quill-mention#364] [Q&A] [answered]
recently upgraded from 3.2.0 to 4.0.0 and the item list shows up displaying the entire html tag

**Comment by csculley** [comment, marked as answer]
Hi, are you returning raw HTML as strings in the `renderItem` or `renderLoading` functions? These will need to be updated with 4.0.0 to return a Node-inheriting element, like this:
https://github.com/quill-mention/quill-mention/blob/6e040106a781711c6040465c26f1f22999e1187a/docs/index.html#L155-L163
